src/lumigo_tracer/lumigo_utils.py

```diff
@@ -66,6 +66,7 @@
 ERROR_SIZE_LIMIT_MULTIPLIER = 2
 CHINA_REGION = "cn-northwest-1"
 EDGE_KINESIS_STREAM_NAME = "prod_trc-inges-edge_edge-kinesis-stream"
+Container = TypeVar("Container", dict, list)
```

Comment:
```suggestion
Container = NewType("Container", Union[List, Dict])
```
IMO :)
https://stackoverflow.com/questions/58755948/what-is-the-difference-between-typevar-and-newtype

Reply:
But `NewType` must be mapped only to one type:
![image](https://user-images.githubusercontent.com/49228804/103170468-1fe95880-484d-11eb-8cca-2a2bee8c1b78.png)

Reply:
`Union[List, Dict]` is a single type 🤔